.vscode/module-docstring.code-snippets

```diff
@@ -0,0 +1,35 @@
+{
+	"Module docstring for mathlib": {
+		"scope": "lean",
+		"prefix": "module docstring",
+		"body": [
+			"/-!",
+            "# ${TM_FILENAME_BASE/([^_]*)(_?)/${1:/capitalize}${2:+ }/g}",
+			"",
+			"## Main definitions",
+			"",
+			"* `foo_bar`",
+			"",
+			"## Main statements",
+			"",
+			"* foo_bar_unique",
+			"",
+			"## Notation",
+			"",
+			"",
+			"",
+			"## Implementation details",
+			"",
+			"",
+			"",
+			"## References",
+			"",
+			"* [F. Bar, *Quuxes*][]",
```

Comment:
```suggestion
			"* [F. Bar, *Quuxes*][bibkey]",
```
or something, to make it clear what goes in the second box?

Reply:
Yeah that sounds good, I made this change and a couple of other tweaks